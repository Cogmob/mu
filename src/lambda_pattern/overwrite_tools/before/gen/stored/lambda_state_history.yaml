current_state: 12345
latest_state: 12345
state_order:
- 12345
states:
  '12345':
    name: old
    lambda_version: 4342434
    dev_built: true
    dev_tests_passed: true
    release_built: true
    release_tests_passed: true
    notes: working
