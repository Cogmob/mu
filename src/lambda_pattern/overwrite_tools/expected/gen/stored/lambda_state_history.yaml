state_version_order:
  - .nan
states:
  NaN:
    name: state
    state_version: a
    dev_built: false
    dev_tests_passed: false
    release_built: false
    release_tests_passed: false
current_state_version: .nan
newest_state_version: .nan
